### PR TITLE
feat: Add fetch countries and jurisditions

### PR DESCRIPTION
**Pull request by raulgomezc**
Also added relationships from officers to officers and modified the structure of the fetch api

Closes [Ticket](https://trello.com)

## Testing

Steps to manually verify the change:
1. _Setup_: step to set the environment
2. _Exercise_: How to run the environment and example link: [Test Link](http://127.0.0.1:8000/)
3. _Verify_: what we need to verify

### CODE REVIEW

File: paradise_papers_search/paradise_papers_search/constants.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from neomodel import db
+
+countries = db.cypher_query("MATCH (n) WHERE NOT n.countries CONTAINS ';' RETURN DISTINCT 'node' as entity, n.countries AS countries UNION ALL MATCH ()-[r]-() WHERE EXISTS(r.countries) RETURN DISTINCT 'relationship' AS entity, r.countries AS countries")[0]
```
Comment (mariamera) [inline paradise_papers_search/paradise_papers_search/constants.py]: this query could be shorter and faster like : `MATCH (n) WHERE NOT n.countries CONTAINS ';' RETURN DISTINCT n.countries AS countries`

File: paradise_papers_search/paradise_papers_search/constants.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+jurisdictions = db.cypher_query("MATCH (n) WHERE EXISTS(n.jurisdiction) RETURN DISTINCT 'node' as entity, n.jurisdiction AS jurisdiction UNION ALL MATCH ()-[r]-() WHERE EXISTS(r.jurisdiction) RETURN DISTINCT 'relationship' AS entity, r.jurisdiction AS jurisdiction")[0]
```
Comment (mariamera) [inline paradise_papers_search/paradise_papers_search/constants.py]: maybe here we could use something like `MATCH (n)  RETURN DISTINCT n.jurisdiction AS jurisdiction`?

File: paradise_papers_search/fetch_api/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+default_app_config = 'fetch_api.apps.FetchApiConfig'
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/__init__.py]: Missing line here

File: paradise_papers_search/fetch_api/models/Officer.py [hunk 4]
```diff
@@ -38,6 +38,23 @@ def serialize_connections(self):
                 'nodes_type': 'entity',
                 'nodes_related': helpers.serialize_relationship(self.entities.all(), 'OFFICER_OF'),
             },
+            {
+                'nodes_type': 'officer',
+                'nodes_related': self._officer_relationships(),
+            },
         ]
 
+    def _officer_relationships(self):
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/models/Officer.py]: This looks like the `entity_relationship` function => https://github.com/neo4j-examples/paradise-papers-django/blob/refactor_app/paradise_papers_search/fetch_api/models/Entity.py#L72
Can we add this to the helper file?

File: paradise_papers_search/fetch_api/views.py [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
 from rest_framework.response import Response
-from rest_framework.renderers import JSONRenderer
-from .models.helpers import count_nodes, fetch_nodes, fetch_node_details
+from .models.helpers import (
+    count_nodes,
+    fetch_nodes,
+    fetch_node_details,
+    fetch_countries,
+    fetch_jurisdictions
+)
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/views.py:9]: 👍

File: paradise_papers_search/fetch_api/views.py [hunk 6]
```diff
@@ -55,5 +58,27 @@ def get(self, request):
                 'data': node_details,
             },
         }
-        json_data = JSONRenderer().render(data)
-        return Response(json_data)
+        return Response(data)
+
+class GetCountries(APIView):
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/views.py:63]: Can we follow the same convention for the function?
Here is a extra line

File: paradise_papers_search/fetch_api/__init__.py (hunk 3 above)
Comment (junior1q94) [inline paradise_papers_search/fetch_api/__init__.py]: Newline here

File: paradise_papers_search/paradise_papers_search/constants.py (hunk 1 above)
Comment (junior1q94) [reply paradise_papers_search/paradise_papers_search/constants.py]: Also let's check if we can format all cypher queries like:
``` python
countries = db.cypher_query(
    """
    MATCH (n) 
    WHERE NOT n.countries CONTAINS ';'
    RETURN DISTINCT n.countries AS countries
    """
)
```

**Review by junior1q94 — CHANGES_REQUESTED** [review]

File: paradise_papers_search/fetch_api/models/helpers.py [hunk 7]
```diff
@@ -64,11 +64,36 @@ def fetch_node_details(node_info):
...
+        nodes   = []
+
+        for row in results[0]:
+            node = self.nodes.get(node_id=row[1])
+            serialized_node = node.serialize
+            serialized_node['relationship'] = row[0].type
+            nodes.append(serialized_node)
+
+
+        return nodes
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/models/helpers.py]: Github will be happy with a extra line here 😸

File: paradise_papers_search/paradise_papers_search/constants.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+jurisdictions = db.cypher_query(
+    '''
+    MATCH (n)
+    RETURN DISTINCT n.jurisdiction AS jurisdiction
+    '''
+)[0]
+
+
+COUNTRIES = sorted([country[0] for country in countries])
+JURISDICTIONS = sorted([jurisdiction[0] for jurisdiction in jurisdictions if isinstance(jurisdiction[0], str) ])
```
Comment (oscarpolanco) [inline paradise_papers_search/paradise_papers_search/constants.py]: Extra space on at the end of the array

**Review by junior1q94 — APPROVED** [review]

**Review by mariamera — APPROVED** [review]
Looks kinda good to me 👍

**Pull request merged**